client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/styles.scss

```diff
@@ -376,7 +376,7 @@
 			}
 			ul {
 				list-style: none;
-				margin-bottom: 48px;
+				margin: 0 0 48px 0;
```

Comment:
Should we scope this to just the FAQ section?

Reply:
It's already scoped to `bulk-domain-transfer__faqs`. 

Currently there are no other `ul` children, so I think we can avoid scoping more at this point of time.